scorecards-site/tailwind.config.js

```diff
@@ -84,6 +84,7 @@ module.exports = {
         'min-w-max',
         'top-100',
         'h-400',
+        'mr-80',
```

Comment:
I was looking at this? Do we have this on the fixed nav bar?

Reply:
Yeah to avoid this clash 
<img width="286" alt="image" src="https://user-images.githubusercontent.com/4757853/158218240-f6db16dd-1fcd-451b-89c9-1ea2d1881e38.png">

Can do it a different way if you'd prefer?